src/structlog/_log_levels.py

```diff
@@ -86,7 +87,8 @@ def exception(self: FilteringBoundLogger, event: str, **kw: Any) -> Any:
 
 
 async def aexception(self: FilteringBoundLogger, event: str, **kw: Any) -> Any:
-    kw.setdefault("exc_info", True)
+    if "exc_info" not in kw:
```

Comment:
Given your explanation on #488, I think this should also handle the case of someone passing `True` here. It works everywhere else, so it's kinda unobvious that it doesn't here.

It would be also nice if you could add a comment explaining what's happening, so I don't "simplify" it in the future. ;)

Reply:
Actually I'm slightly puzzled about the expected behavior of this call.

Synchronous `exception()` either passes `exc_info` from `kwargs` as is, or sets it to `True` (which is replaced with actual exception info later on).  

For async variant, my assumption was to put use `sys.exc_info()` for default case, and pass everything else as is. Indeed, it makes explicit `aexception(...., exc_inf=True)` not working as expected.

However - should I replace literal `True` with `exc_info`? Replacing any true-ish value doesn't seem right, either.

Reply:
yes, that's what I meant: replace `True` (and only `True`) by `sys.exc_info()` in addition to your logic.

I.e. something like `if kw.get("exc_info", True) is True:`:

```pycon
>>> {}.get("exc_info", True) is True
True
>>> {"exc_info": True}.get("exc_info", True) is True
True
>>> {"exc_info": (1,2,3)}.get("exc_info", True) is True
False
```

In vanilla `exception()`, passing True the way to tell structlog to figure it out, but as you've pointed out yourself: that's impossible once we're in a new task so we have to handle it here.

Reply:
Ok, makes sense. Update pushed.